evm/src/cpu/kernel/txn_fields.rs

```diff
@@ -0,0 +1,59 @@
+/// These are normalized transaction fields, i.e. not specific to any transaction type.
+#[allow(dead_code)]
+#[derive(Copy, Clone, Eq, PartialEq, Hash, Ord, PartialOrd, Debug)]
+pub(crate) enum NormalizedTxnField {
```

Comment:
Don't we also need the access list here?

Reply:
Hm my intention was these fields would cover single-word values. Maybe we should store some kind of length field here like `DataLen`, but I'd prefer to punt on it for now, as I'm not sure what the best way to store access list would be (simple list, hash map, etc).